feat: contributing (#4132)
OI-wiki/OI-wiki: contributing.md

## OI Wiki 贡献指南

:+1::tada: 首先，感谢您抽出宝贵时间为本项目做出贡献！ :tada::+1:

以下是为 OI Wiki 做出贡献的指南，这些指南托管在 GitHub 上的 [OI Wiki 组织](https://github.com/oi-wiki/) 中。

> 若您认为此文档有不尽人意之处，请随时在拉取请求中加入对本文档的更改。

Reviewer: ```suggestion
若您认为此文档有不尽人意之处，请随时发起拉取请求完善本文档。
```
也是机翻？偏离原意并且生硬

> 若您认为此文档有不尽人意之处，请随时在拉取请求中加入对本文档的更改。

Reviewer 2: ```suggestion
若您认为此文档有不尽人意之处，欢迎在拉取请求中加入对本文档的更改。
```

#### 目录

> [CoC](#CoC)
> 
> [如何参与](#如何参与)
> 
> [我需要遵循什么？](#我需要遵循什么)
> 
> ## CoC
> 
> 您可以在 [此处](CODE_OF_CONDUCT.md) 找到 OI Wiki 的 Code of Conduct。

Reviewer: ```suggestion
[行为准则](#行为准则)

[如何参与](#如何参与)

[我需要遵循什么？](#我需要遵循什么)

## 行为准则

您可以在 [此处](CODE_OF_CONDUCT.md) 找到 OI Wiki 的行为准则。
```
作为一项建议，我觉得使用对应的中文翻译「行为准则」更好，更能直接表达意思
同时，如上提到的，我觉得我们如果有一份中文版本的行为准则会更好

## 如何参与

> 您可以在 [HTC - OI Wiki](https://oi-wiki.org/intro/htc/) 中查看有关如何参与的详细信息。

Reviewer: ```suggestion
您可以在 [How to contribute](https://oi-wiki.org/intro/htc/) 中查看有关如何参与的详细信息。
```
使用全称也许更好

此外，请阅读 [格式指南](https://oi-wiki.org/intro/format/) 中的相关内容。

## 我需要遵循什么？

> 如果您发现有人不遵守此处的指南，请随时联系 [hi@oi-wiki.org](mailto:hi@oi-wiki.org)。

Reviewer 2: ```suggestion
如果您发现有人不遵守此处的指南，欢迎联系 [hi@oi-wiki.org](mailto:hi@oi-wiki.org)。
```

> + 我们欢迎您审核其他拉取请求，但请以友好的方式发表评论。此类负面评论会深深伤害社区贡献者的贡献热情，因此不建议这样做。当你在拉取请求中发现问题时，欢迎进一步的拉取请求，但不欢迎你对贡献者（拉取请求作者）发表负面评价。

Reviewer: ```suggestion
+ 我们欢迎您审核其他拉取请求，但请以友好的方式发表评论。负面评论会深深伤害社区贡献者的贡献热情，因此不建议这样做。当你在拉取请求中发现问题时，欢迎进一步的拉取请求，但不欢迎你对贡献者（拉取请求作者）发表负面评价。
```
没有上文提到何谓「此类」，应为笔误？

+ 请在评论问题或拉取请求之前仔细阅读，以免引起不必要的误会。

+ 建议跟进您的拉取请求。请不要在新建一个拉取请求后不管不顾。它会让 OI Wiki 编辑和管理员非常头疼（找不到原作者等）。

> + 请记住，您在一个社区中，您需要学会接受其他人的贡献，以及他们贡献的内容，无论质量高低，甚至可能和他们协作。如果您不同意这个观点，请随时 fork 这个仓库并自己进行更改。另一方面，你的想法可能不够完善，所以要听取别人的意见。

Reviewer 2: ```suggestion
+ 请记住，您在一个社区中，您需要学会接受其他人的贡献，以及他们贡献的内容，无论质量高低，甚至可能和他们协作。如果您不同意这个观点，您可以 fork 这个仓库并自己进行更改。另一方面，你的想法可能不够完善，所以要听取别人的意见。
```

> 通过为社区做出贡献，您同意 OI Wiki 组织可以禁止违反社区准则并破坏社区和谐的用户访问组织。

Reviewer: ```suggestion
当您为社区做出贡献时，您亦同意 OI Wiki 组织可以禁止违反社区准则并破坏社区和谐的用户访问组织。
```
有点生硬，感觉像是机翻

# Chinese
## OI Wiki 贡献指南

:+1::tada: 首先，感谢您抽出宝贵时间为本项目做出贡献！ :tada::+1:

以下是为 OI Wiki 做出贡献的指南，这些指南托管在 GitHub 上的 [OI Wiki 组织](https://github.com/oi-wiki/) 中。

若您认为此文档有不尽人意之处，请随时在拉取请求中加入对本文档的更改。

#### 目录

[CoC](#CoC)

[如何参与](#如何参与)

[我需要遵循什么？](#我需要遵循什么)

## CoC

您可以在 [此处](CODE_OF_CONDUCT.md) 找到 OI Wiki 的 Code of Conduct。

## 如何参与

您可以在 [HTC - OI Wiki](https://oi-wiki.org/intro/htc/) 中查看有关如何参与的详细信息。

此外，请阅读 [格式指南](https://oi-wiki.org/intro/format/) 中的相关内容。

## 我需要遵循什么？

> 如果您发现有人不遵守此处的指南，请随时贡献自己或联系 [hi@oi-wiki.org](mailto:hi@oi-wiki.org)。

Reviewer 3: 这句怎么是机翻（

但我有点没看懂，为什么发现别人不遵守指南要自己贡献，是贡献什么？

> + 我们欢迎您审核其他拉取请求，但请以友好的方式发表评论。此类负面评论会深深伤害社区贡献者的贡献热情，因此不建议这样做。当你在拉取请求中发现问题时，欢迎进一步的拉取请求，但不欢迎你在你在这个拉取请求中没有建设性的情况下对贡献者（拉取请求作者）发表负面评价。

Reviewer 4: ```suggestion
+ 我们欢迎您审核其他拉取请求，但请以友好的方式发表评论。此类负面评论会深深伤害社区贡献者的贡献热情，因此不建议这样做。当你在拉取请求中发现问题时，欢迎进一步的拉取请求，但不欢迎你对贡献者（拉取请求作者）发表负面评价。
```

有贡献也不能乱喷（（

+ 请在评论问题或拉取请求之前仔细阅读，以免引起不必要的误会。

> + 建议跟进您的拉取请求。请不要在新建一个拉取请求后不管不顾。它会让 OI Wiki 编辑和管理员非常忙碌（找不到原作者等）。

Reviewer 5: ```suggestion
+ 建议跟进您的拉取请求。请不要在新建一个拉取请求后不管不顾。它会让 OI Wiki 编辑和管理员非常头疼（找不到原作者等）。
```
好像也不会很忙（？）主要是比较头疼

> + 请记住，您在一个社区中，您需要学会接受其他人的贡献，以及他们贡献的内容，无论质量高低，甚至可能和他们协作。如果您不同意这个观点，请随时 fork 这个仓库并自己进行更改。另一方面，你的想法可能不如别人的好，所以要吸取别人的意见。

Reviewer: ```suggestion
+ 请记住，您在一个社区中，您需要学会接受其他人的贡献，以及他们贡献的内容，无论质量高低，甚至可能和他们协作。如果您不同意这个观点，请随时 fork 这个仓库并自己进行更改。另一方面，你的想法可能不够完善，所以要听取别人的意见。
```
「不如别人的好」语气太重，「吸取」词语搭配不当

> + 请记住，您在一个社区中，您需要学会接受其他人的贡献，以及他们贡献的内容，无论质量高低，甚至可能和他们协作。如果您不同意这个观点，请随时 fork 这个仓库并自己进行更改。另一方面，你的想法可能不如别人的好，所以要吸取别人的意见。

Reviewer 5: ```suggestion
+ 请记住，您在一个社区中，您需要学会接受其他人的贡献，以及他们贡献的内容，无论质量高低，甚至可能和他们协作。如果您不同意这个观点，请随时 fork 这个仓库并自己进行更改。另一方面，你的想法可能不如别人的好，所以建议尝试吸取别人的意见。
```
要 听起来有些语气强烈了，是我的错觉吗🤔

## Code of Conduct

You can find the OI Wiki Code of Conduct [here](CODE_OF_CONDUCT.md).

## How Can I Contribute?

You can see details on how to contribute in [HTC - OI Wiki](https://oi-wiki.org/intro/htc/).

Also, please read guidelines on format in the [Format Guide](https://oi-wiki.org/intro/format/).

## What do I Need to Follow?

If you find someone not following the guidelines here, feel free to contribute yourself or contact [hi@oi-wiki.org](mailto:hi@oi-wiki.org).

+ We welcome you to review other pull requests, but please comment in a friendly way. Such comments that cast negative reviews will deeply hurt the community contributors' feelings, and therefore is not recommended to do so. When you find a problem in a pull request, a further pull request is welcome, but it is not welcome to speak negatively of the contributors (pull request authors), while you have no constructiveness in this pull request.

+ Please read carefully before commenting on an issue or a pull request.

+ It is recommended to keep track with your pull request. Please don't open a pull request, and stop keeping track. It will keep the OI Wiki editors and maintainers busy.

> + Remember that you are in a community, and you need to accept others to contribute, along with their contributed content, whether it is with quality or not, maybe even collaborate. If you cannot agree with this concept, feel free to fork this repository and make your changes by your own. On the other hand, your ideas might not be as good as others, so listen to other.

Reviewer: ```suggestion
+ Remember that you are in a community, and you need to accept others to contribute, along with their contributed content, whether it is with quality or not, maybe even collaborate. If you cannot agree with this concept, feel free to fork this repository and make your changes by your own. On the other hand, your ideas might not be perfect, so listen to other.
```
同上
